src/math/TransformBoundedField.cc

```diff
@@ -147,13 +147,11 @@ bool TransformBoundedField::operator==(BoundedField const& rhs) const {
     if (!rhsCasted) return false;
 
     return getBBox() == rhsCasted->getBBox() &&
-           *(getTransform().getFrameSet()) == *(rhsCasted->getTransform().getFrameSet());
+           *(getTransform().getMapping()) == *(rhsCasted->getTransform().getMapping());
 }
 
 std::string TransformBoundedField::toString() const {
-    std::ostringstream os;
-    os << "TransformBoundedField (containing " << _transform.getFrameSet()->getNFrame() << " frames)";
-    return os.str();
+    return "TransformBoundedField";
```

Comment:
I don't know if there's a better way to get some amount more description in this returned string?

Reply:
I have not thought of one